docs/SDK/Explanation/litActions.md

```diff
@@ -160,6 +160,9 @@ runLitAction();
 
 Lit Actions inherit the powerful condition checking that Lit Protocol utilizes for Access Control. You can easily check any on-chain condition inside a Lit Action. For example, the below Lit Action will check if the user has at least 1 Wei on Ethereum, and only sign if they do.
 
+<iframe frameborder="100px" width="100%" height="500px" src="https://replit.com/@lit/Lit-Actions-Conditional-Signing/#lit-actions_conditional_signing/src/App.js"></iframe>
```

Comment:
This Replit example doesn't match the description above. Could put this in its own section on this page, description for it is [here](https://deploy-preview-36--lit-dev-docs.netlify.app/coreConcepts/LitActionsAndPKPs/toolsAndExamples#conditional-signing)